fideslog/sdk/python/__init__.py

```diff
@@ -1,3 +1,3 @@
 from . import _version
 
-__version__ = _version.get_versions()["version"]
+__version__ = _version.get_versions()["version"]  # type: ignore
```

Comment:
needed to get around this `mypy` err: `fideslog/sdk/python/__init__.py:3: error: Module has no attribute "get_versions"  [attr-defined]`

Reply:
If you bump the `# type: ignore` to the top of the file, it would apply to the whole file and prevent us from needing to make this change later (if more good, but type-unsafe code is added).

Reply:
this didn't actually work, and caused this mypy err: 
```
fideslog/sdk/python/client.py:14: error: Module "fideslog.sdk.python" has no attribute "__version__"  [attr-defined]
    from . import __version__
    ^
Found 1 error in 1 file (checked 24 source files)
```

So I've left as is for now

Reply:
Weird...this has definitely worked in other projects. Whatever, it's not a huge deal.